apps/ecs/fields/index.js

```diff
@@ -157,5 +157,32 @@ module.exports = {
         value: 'no'
       }
     ]
+  },
+  'job-title': {
+    validate: ['required', 'notUrl']
+  },
+  'hours-of-work-per-week': {
+    validate: ['required', { type: 'max', arguments: 99 }, { type: 'min', arguments: 1 }],
+    attributes: [{ suffix: 'hours' }],
+    className: 'govuk-input govuk-!-width-one-quarter'
```

Comment:
This classname can be for this field - 'govuk-input--width-2' - to align with the figma design.